content/en/docs/instrumentation/swift/_index.md

```diff
@@ -1,7 +1,9 @@
 ---
 title: Swift
 weight: 28
-description: A language-specific implementation of OpenTelemetry in Swift.
+description: >-
+  <img width="35" src="https://github.com/open-telemetry/opentelemetry.io/blob/main/iconography/32x32/Swift.svg"></img>
```

Comment:
Update with URL to raw.github:

https://raw.githubusercontent.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Swift.svg

Reply:
Done